docs/install/supported.rst

```diff
@@ -4,22 +4,12 @@
 Supported Platforms
 ^^^^^^^^^^^^^^^^^^^
 
-PyNE is currently built and tested daily on the following platforms
-on Python 2.7
+PyNE is currently built and tested on the following platforms
+on Python 3.6, 3.7, and 3.8:
 
- * Ubuntu 12.04 - x86_64
- * Ubuntu 10.04 - x86_64
- * Scientific Linux 6 - x86_64
- * OSX 10.8 - x86_64
+ * Ubuntu 16.04 - x86_64
+ * Ubuntu 18.04 - x86_64
 
```

Comment:
We can also add Ubuntu 20.04 - x86_64 here as a supported platform. I installed pyNE from source and ubuntu_18.04.sh file. It is working with no problem.